packages/drivers/routerlicious-driver/src/test/wholeSummaryDocumentStorageService.spec.ts

```diff
@@ -142,6 +142,7 @@ describe("WholeSummaryDocumentStorageService", () => {
             "id",
             new MockGitManager() as any,
             DebugLogger.create("fluid:testSummaries"),
+            { maximumCacheDurationMs: 0 },
```

Comment:
This makes me wonder if we can instead institute 0 as a default value, rather than making it required.

Reply:
For posterity:  This would be a BAD idea because then every driver by default would be lying until they put in some concerted work - they'd project this policy but have no accountability to actually implement it.